unreleased.rst

```diff
@@ -13,12 +13,8 @@ HEAD — ongoing
   file to be a pointer to the source directory of the DAML code contained in a project relative to
   the project root. This is breaking projects, where the ``source`` field of the project is pointing
   to a non-toplevel location in the source code directory structure.
-+ [Sandbox] The sandbox now properly sets the connection pool properties ``minimumIdle``, ``maximumPoolSize``, and ``connectionTimeout``.
```

Comment:
Should these lines really be deleted?

Reply:
yes, a previous automerge messed up the release notes.

Reply:
Ah now I see, these are notes that were already moved to the actual release notes. :+1: